examples/tutorials/plot_tuto_mcar_test.py

```diff
@@ -0,0 +1,149 @@
+"""
+============================================
+Tutorial for testing the MCAR case
+============================================
+
+In this tutorial, we show how to use the mcar test classe and it methods
+
+Keep in my mind that, at this moment, the mcar tests are only handle tabular data.
+"""
+# %%
+# First import some libraries
+from matplotlib import pyplot as plt
+import random
+
+import numpy as np
+import pandas as pd
+
+from qolmat.audit.holes_characterization import MCARTest
+
+# %%
+# 1. The Little's test
+# ---------------------------------------------------------------
+# How to use the Little's test ?
+# ==============================
+# When we deal with missing data in our dataset it's interesting to know the nature of these holes.
+# There exist three types of holes : MCAR, MAR and MNAR.
+# (see the: `Rubin's missing mechanism classification
+# <https://qolmat.readthedocs.io/en/latest/explanation.html>`_)
+#
+# The simplest case to test is the MCAR case. The most famous MCAR statistical test is the
+# `Little's test <https://www.tandfonline.com/doi/abs/10.1080/01621459.1988.10478722>`_.
+# Keep in mind that the Little's test is designed to test the homogeneity of means between the
+# missing patterns and won't be efficient to detect the heterogeneity of covariance between missing
+# patterns.
+#
+# This notebook shows how the Little's test performs and its limitations.
+
```

Comment:
Je pense qu'il faudrait bien précisé dans un test qu'on présente en tuto les hypothèses H0 et H1 et le seuils (1.96, signifie 5% si je ne me trompe pas) :)

Reply:
*préciser on peut faire des review des review ?

Keep in my mind that, at this moment, the mcar tests [are] only handle tabular data.